packages/atlas/src/views/viewer/ChannelView/ChannelViewTabs/ChannelNfts.tsx

```diff
@@ -38,6 +40,9 @@ export const ChannelNfts: React.FC<ChannelNftsProps> = ({
       creatorChannel: {
         id_eq: channelId,
       },
+      video: {
+        isPublic_eq: activeChannelId !== channelId ? true : undefined,
```

Comment:
Would it be a lot of hassle to actually check if the user is the owner of the channel instead of checking if the users is logged in as a channel? That would help in cases where I own multiple channels (but I can only be "logged in" as a single channel at a given time)